ibm_catalog.json

```diff
@@ -222,12 +222,15 @@
             },
             {
               "key": "strategy_type",
+              "type_metadata": "string",
               "options": [
                 {
+                  "description": "Build using a Dockerfile in your repository.",
                   "displayname": "Dockerfile",
                   "value": "dockerfile"
                 },
                 {
+                  "description": "Automatically detect and build using Cloud Native Buildpacks.",
```

Comment:
here as well, can we add learn more pointing to [buildpack strategy](https://cloud.ibm.com/docs/codeengine?topic=codeengine-plan-build#build-buildpack-strat) ?

can we also add what detects automatically?

something like 
>Automatically detects the runtime and builds a container image from source using Paketo Buildpacks.